packages/server/src/api/routes/tests/rowAction.spec.ts

```diff
@@ -129,9 +132,24 @@ describe("/rowsActions", () => {
 
       expect(await config.api.rowAction.find(tableId)).toEqual({
         actions: {
-          [responses[0].id]: { ...rowActions[0], id: responses[0].id, tableId },
-          [responses[1].id]: { ...rowActions[1], id: responses[1].id, tableId },
-          [responses[2].id]: { ...rowActions[2], id: responses[2].id, tableId },
+          [responses[0].id]: {
+            name: rowActions[0].name,
+            id: responses[0].id,
+            tableId,
+            automationId: expect.any(String),
```

Comment:
```suggestion
            automationId: expectAutomationId(),
```

For this and the 2 below.